modules/container/src/main/resources/META-INF/mule-test-module.properties

```diff
@@ -0,0 +1,3 @@
+module.name=container
+
+artifact.export.classPackages=org.mule.runtime.container.internal
```

Comment:
?

Reply:
MuleArtifactFunctionalTestCase extends FunctionalTestCase which has to access these clases in order to reuse the code for ModuleRepository and just change the ClasspathModuleDiscoverer temporary folder logic. This is a mechanism that has the runner to export internal classes for test only.